src/main/java/pl/targosz/homework/TextOperator.java

```diff
@@ -0,0 +1,34 @@
+package pl.targosz.homework;
+
+public class TextOperator {
+    public static boolean isPalindrome(String text) {
+        int length = text.length();
+        int forward = 0, backward = length - 1;
+        while (backward > forward) {
+            char forwardChar = text.charAt(forward++);
+            char backwardChar = text.charAt(backward--);
+            if (forwardChar != backwardChar)
+                return false;
+
+
+        }
+        return true;
+    }
+
+    public static void main(String[] args) {
+        String text = "Sigib";
+        String[] someArray = text.split("");
+        for (int i = 0; i < someArray.length; i++) {
```

Comment:
To jest przekombinowane.

"sprawdzić, czy tekst zawiera jakiekolwiek znaki" - to można zrobić jedną metodą (sprawdź jakie metody dostarcza String).

"jeśli punkt 1 jest prawdziwy, to obliczyć długość tekstu"

A to gdzie?

No i najważniejsze - przeczytaj dokładnie co powinien program wykonać:
![image](https://user-images.githubusercontent.com/17989854/137578899-6ad6f9ed-ef3c-4dad-b860-009cfa4286d0.png)